Signum.Utilities/Signum.Utilities.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <Nullable>enable</Nullable>
-    <WarningsAsErrors>nullable</WarningsAsErrors>
+    <!--<WarningsAsErrors>nullable</WarningsAsErrors>-->

```

Comment:
?

Reply:
reverted in b0839acd593fe3d42436a1a10f1bd6cd4b04b57d.